sqlsimplekv/keyvalue.go

```diff
@@ -0,0 +1,214 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package sqlsimplekv
+
+import (
+	"database/sql"
+	"time"
+
+	sqldriver "database/sql/driver"
+
+	"golang.org/x/net/context"
+	errgo "gopkg.in/errgo.v1"
+
+	"github.com/juju/simplekv"
+	"github.com/lxc/lxd/shared/logger"
```

Comment:
I'm pretty sure that's not the logger you want.

Reply:
good point. I removed the log message.